processes-webapp/src/main/webapp/vue-app/processes/components/WorkFlowList.vue

```diff
@@ -51,7 +52,7 @@
             prepend-inner-icon="mdi-filter" />
         </v-col>
       </v-row>
-      <v-row v-if="isXSmall">
+      <v-row v-if="isXSmall && workflowList.length>0">
```

Comment:
Can you add  another computed variable for `this.workflowList?.length`  with the name emptyWorkflow so that the code can be well understood